packages/suite/src/components/suite/Preloader/components/InitialLoading/index.tsx

```diff
@@ -1,5 +1,10 @@
 import React, { useEffect, useState } from 'react';
 import { Loading } from '@suite-components';
+import styled from 'styled-components';
+
+const StyledLoading = styled(props => <Loading {...props} />)`
```

Comment:
```
const StyledLoading = styled(Loading)`
```
should be enough unless you run into one bug and in order to resolve it you really needed to spread props like that. I am nitpicking this just to make sure.

Reply:
There are few other places like this, I am gonna keep only this one comment so it won't get too messy

Reply:
I added this, because I had problems with circular dependencies and the build didn't work :( I found this "fix" in some other places...

Reply:
yup I was afraid this was the case, no problem here then :)